public/js/main.js

```diff
@@ -6,8 +6,8 @@ function dateClickHandler (e) {
     date_to: addDays(date, 14)
   };
   getEventsModule(dateParams, renderModule);
-  document.getElementById('list-page-content').classList.toggle('hide');
-  document.getElementById('show-cal').classList.toggle('hide-cal');
+  document.getElementById('list-page-content').style.display = 'block';
+  document.getElementById('show-cal').style.display = 'none';
```

Comment:
These should still both be classList toggles really - better to keep JS and CSS styling as separate as possible, and it's easier to check for a class than check the style of an element

Reply:
we got it back the way it was but it's not working.
there is no such class called hide

Reply:
Yea you need to make one :p